src/nidaqmx/_task_modules/channels/ai_channel.py

```diff
@@ -3,43 +3,90 @@
 import ctypes
```

Comment:
Reformatting *everything* is overdoing it. Please specify exclusions in the `[tool.black]` section of `pyproject.toml`:
- All code-generated files (both in src/ and generated/). You might need to list out individidual files.
- All metadata files.

Once the new codegen is done, we can _consider_ using `black` to reformat the output, but until then we shouldn't bother because it will just make it harder to compare codegen.